openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -69,20 +70,16 @@ <h2 class="hd hd-3 page-title">${course.display_name_with_default}</h2>
                 <div class="section section-tools">
                     <h3 class="hd-6">${_("Course Tools")}</h3>
                     <ul class="list-unstyled">
-                        <li>
-                            <a href="${reverse('openedx.course_bookmarks.home', args=[course_key])}">
-                                <span class="icon fa fa-bookmark" aria-hidden="true"></span>
-                                ${_("Bookmarks")}
-                            </a>
-                        </li>
-                        % if UNIFIED_COURSE_TAB_FLAG.is_enabled(course.id):
-                            <li>
-                                <a href="${reverse('openedx.course_experience.course_updates', args=[course.id])}">
-                                    <span class="icon fa fa-newspaper-o" aria-hidden="true"></span>
-                                    ${_("Updates")}
-                                </a>
-                            </li>
-                        % endif
+                        % for course_tool in CourseToolsPluginManager.get_course_tools():
```

Comment:
Do you want to handle the case where a course has no applicable Course Tools and you drop the section entirely (or have a no tools message)?